src/utils/roleAssignment.ts

```diff
@@ -0,0 +1,44 @@
+import { UnifiedRoleAssignment } from "@microsoft/microsoft-graph-types";
+import request, { CliRequestOptions } from "../request.js";
+
+const graphResource = 'https://graph.microsoft.com';
+
+/**
+ * Utils for RBAC.
+ * Supported RBAC providers:
+ *  - Directory (Entra ID)
+ */
+export const roleAssignment = {
+  /**
+   * Assigns a specific role to a principal and defines a set of resources (scope) that the access applies to.
+   * Directory (Entra ID) RBAC provider
+   * @param roleDefinitionId Role which lists the actions that can be performed
+   * @param principalId Object that represents a user, group, service principal, or managed identity that is requesting access to resources
+   * @param directoryScopeId Set of resources that the access applies to
+   * @returns
+   */
+  async createEntraIDRoleAssignment(roleDefinitionId: string, principalId: string, directoryScopeId: string): Promise<UnifiedRoleAssignment> {
```

Comment:
Should this function be exported since we've got two overloads that call it and which seem like what we want command builders to use?

Reply:
Refactored to export only two functions